src/main/scala/com/fulcrumgenomics/fastq/FastqToBam.scala

```diff
@@ -92,6 +92,7 @@ class FastqToBam
   @arg(flag='s', doc="If true, queryname sort the BAM file, otherwise preserve input order.")  val sort: Boolean = false,
   @arg(flag='u', doc="Tag in which to store molecular barcodes/UMIs.")                         val umiTag: String = ConsensusTags.UmiBases,
   @arg(flag='q', doc="Tag in which to store molecular barcode/UMI qualities.")                 val umiQualTag: Option[String] = None,
+  @arg(flag='b', doc="Tag in which to store sample barcode qualities.")                        val smpQualTag: Option[String] = None,
```

Comment:
since we have a short flag, it's not the worst to have a fully qualified long name 

```suggestion
  @arg(flag='b', doc="Tag in which to store sample barcode qualities.")                        val sampleBarcodeQualTag: Option[String] = None,
```

Reply:
What do you thin about just making this a `boolean` value, and then hard code `QT` below (like we do for `BC`) since thats the recommended tag from [the SAM spec](https://samtools.github.io/hts-specs/SAMtags.pdf)?